src/GslibIO.jl

```diff
@@ -155,4 +202,58 @@ function save(file::File{format"GSLIB"}, sdata::AbstractData)
        propnames=vars)
 end
 
+# low level function for saving `data` to a legacy GSLIB format using `varnames` as variable names
+function save_legacy(filename::AbstractString, data::AbstractMatrix, varnames::NTuple; 
+                     na=-999.0, header="This file was generated with GslibIO.jl")
```

Comment:
For the save_legacy function we need to consume GeoStats.jl objects directly. This means that the function should receive an `AbstractData` from GeoStatsBase.jl. Most options will be not needed given that the spatial data types have all information in them. Can you please refactor the save_legacy accordingly? Take a look at the `save` implementation.

Reply:
Let's discuss this after unit tests for load_legacy